test/OptionSettlementEngine.t.sol

```diff
@@ -805,15 +818,15 @@ contract OptionSettlementTest is Test, NFTreceiver {
         vm.prank(ALICE);
         uint256 cTokenId2 = engine.write(oTokenId, 3);
 
-        (uint160 decodedOptionIdFromOTokenId, uint96 decodedClaimIndexFromOTokenId) = engine.decodeTokenId(oTokenId);
+        (uint160 decodedOptionIdFromOTokenId, uint96 decodedClaimIndexFromOTokenId) = decodeTokenId(oTokenId);
```

Comment:
Ditto for all these — should be xyzOptionKey and xyzClaimKey

Reply:
Will hit in the test cleanup PR.